tests/scripts/watch_incorrect_protobuf.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo '"does not match ContainerLimitation"' >$2
```

Comment:
This is supposed to test what happens if the JSON is not matching the model (JSON is valid but one key is incorrect for instance) but in this case you're doing the same test as the malformed one beneath (where you are testing the output is not a valid JSON).

Reply:
I'm not sure to understand: my `watch_malformed.sh` fixture outputs malformed JSON.
```echo 'MALFORMED' >$2```

The `watch_incorrect_protobuf.sh` mentioned above outputs correct JSON but not matching the `ContainerLimitation` protobuf model.
So I think I test both cases, even though they go through the exact same code path (inside `watch()`).

On the other side, there are still a few code paths not tested, I'll fix that.

Reply:
the malformed test is correct, the incorrect protobuf should be a valid json with wrong model.

Reply:
This is the case. [A string is a valid JSON](https://www.json.org/).

Reply:
Got it. I hope they follow the standard :).